README.rst

```diff
@@ -10,6 +10,16 @@ about writing one, please start reading at :pep:`1`. Note that the PEP Index
 (:pep:`0`) is automatically generated based on the metadata headers in other PEPs.
 
 
+Canonical links
+===============
+
+The canonical form of PEP links are zero-padded, such as
+https://peps.python.org/pep-0008/
```

Comment:
```suggestion
``https://peps.python.org/pep-0008/``
```

Since this is primarily included to be a literal rather than for its value as a link.